src/web/app/pages-help/instructor-help-page/instructor-help-page.component.html

```diff
@@ -1,49 +1,70 @@
-<div>
-  <h1 class="color-orange"> Help for Instructors</h1>
-  <div>
-    <p>
-      Have questions about how to use TEAMMATES? This page answers some frequently asked questions.
-      <br> If you are new to TEAMMATES, our <a routerLink="../getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
-      <br> If you have any remaining questions, don't hesitate to <a href="mailto:{{ supportEmail }}">email us</a>. We respond within 24 hours.
-    </p>
-    <div id="topics" style="display: block;">
+<div class="main-body" #helpPage>
+  <div id="body">
+    <h1 class="color-orange" id="body"> Help for Instructors</h1>
+    <div>
       <p>
-        Browse questions by topic:
+        Have questions about how to use TEAMMATES? This page answers some frequently asked questions.
+        <br> If you are new to TEAMMATES, our <a routerLink="../getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
+        <br> If you have any remaining questions, don't hesitate to <a href="mailto:{{ supportEmail }}">email us</a>. We respond within 24 hours.
       </p>
-      <ul>
-        <li>
-          <a href="#students">Students</a>
-        </li>
-        <li>
-          <a href="#courses">Courses</a>
-        </li>
-        <li>
-          <a href="#sessions">Sessions</a>
-        </li>
-        <li>
-          <a href="#questions">Questions</a>
-        </li>
-      </ul>
-    </div>
-  </div>
-  <div class="row">
-    <div class="offset-md-2 col-md-8">
-      <div class="input-group input-group-lg">
-        <input type="text"  [(ngModel)]="searchTerm" class="form-control input-lg" placeholder="How can we help?" (keyup.enter)="search()">
-        <button class="btn btn-primary btn-lg" (click)="search()"><i class="fas fa-search"></i></button>
-        <button class="btn btn-danger btn-lg" (click)="clear()"><i class="fas fa-times"></i></button>
+      <div class="row">
+        <div class="offset-md-2 col-md-8">
+          <div class="input-group input-group-lg">
+            <input type="text" [(ngModel)]="searchTerm" class="form-control input-lg" placeholder="How can we help?" (keyup.enter)="search()">
```

Comment:
[Migrating to v4](https://getbootstrap.com/docs/4.0/migration/#forms-1):
`Renamed .input-lg and .input-sm to .form-control-lg and .form-control-sm, respectively.`

Reply:
These lines weren't added by me. They were originally in line 32, but the shuffling around has made Git think I wrote them. I'd be happy to refactor them in a different pull request, though.

Reply:
Refactoring small stuff like this in another PR would be too insignificant. Moreover, it is just 1 class that is affected and your changes in this PR overlaps with the issue at hand.

However, if there are a lot of these issues in this page, then it is justifiable to do it in another PR.

Reply:
Gotcha